optimizely/optimizely.py

```diff
@@ -40,6 +41,7 @@ def __init__(
         skip_json_validation=False,
         user_profile_service=None,
         sdk_key=None,
+        access_token=None,
```

Comment:
This will break the SDK, since a user can choose not to use named arguments, but rather pass them in as positional arguments. This should be the last argument in the method.

Also, I am not sure if `access_token` needs to be introduced here. It is a concept in the config manager and one can simple invoke the `AuthDatafilePollingConfigManager` and pass that in. So this is probably not useful.

Reply:
good catch on the order of arguments, wondering how it passed the unit tests. 

IMO, adding access_token here  as the last optional argument does not hurt and even makes sense. Each config manager (Polling and Static) with default parameters is referenced in here, so not sure why AuthDatafilePollingConfigManager has to be the exception. If access_token was set on PollingConfigManager then this could be considered as an option just like poling interval, but AuthDatafilePollingConfigManager is a separate entity.

Reply:
Done. Though, I think that `access_token` should still be included in the constructor in order to use `AuthDatafilePollingConfigManager` in the case where someone chooses to create an `optimizely` instance.